sensu/gohandler_test.go

```diff
@@ -99,7 +102,8 @@ func goHandlerExecuteUtil(t *testing.T, handlerConfig *PluginConfig, eventFile s
 	values := handlerValues{}
 	options := getHandlerOptions(&values)
 
-	goHandler := NewGoHandler(handlerConfig, options, validationFunction, executeFunction)
+	goPlugin := NewGoHandler(handlerConfig, options, validationFunction, executeFunction)
+	goHandler := goPlugin.(*goHandler)
```

Comment:
Type assertion not necessary.

Reply:
See previous comment.

Reply:
Removed code assertion since NewGoHandler now returns a GoHandler.